From 1cc0b7233ca7413a2c21d35efe58d0fa879804e3 Mon Sep 17 00:00:00 2001
From: Luiz Capitulino <lcapitulino@redhat.com>
Date: Wed, 30 Jun 2010 16:29:56 -0300
Subject: [PATCH 1/9] doc: Update monitor info subcommands

RH-Author: Luiz Capitulino <lcapitulino@redhat.com>
Message-id: <1277915402-6415-2-git-send-email-lcapitulino@redhat.com>
Patchwork-id: 10379
O-Subject: [PATCH 1/7] doc: Update monitor info subcommands
Bugzilla: 582262
RH-Acked-by: Kevin Wolf <kwolf@redhat.com>
RH-Acked-by: Markus Armbruster <armbru@redhat.com>
RH-Acked-by: Juan Quintela <quintela@redhat.com>
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>

From: Stefan Hajnoczi <stefanha@linux.vnet.ibm.com>

The "info blockstats" documentation was copy-pasted as "info block"
instead of "info blockstats".  The documentation for "commands", "jit",
"numa", "qdm", and "roms" is missing.  This patch resolves these issues
in qemu-monitor.hx.

Signed-off-by: Stefan Hajnoczi <stefanha@linux.vnet.ibm.com>
Signed-off-by: Aurelien Jarno <aurelien@aurel32.net>
(cherry picked from commit d8e06f973cc8e42aa1faa1216592aa9eb671f360)
---
 qemu-monitor.hx |   12 +++++++++++-
 1 files changed, 11 insertions(+), 1 deletions(-)

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 qemu-monitor.hx |   12 +++++++++++-
 1 files changed, 11 insertions(+), 1 deletions(-)

diff --git a/qemu-monitor.hx b/qemu-monitor.hx
index f7d08b0..6bc60fb 100644
--- a/qemu-monitor.hx
+++ b/qemu-monitor.hx
@@ -54,13 +54,15 @@ Show various information about the system state.
 @table @option
 @item info version
 show the version of QEMU
+@item info commands
+list QMP available commands
 @item info network
 show the various VLANs and the associated devices
 @item info chardev
 show the character devices
 @item info block
 show the block devices
-@item info block
+@item info blockstats
 show block device statistics
 @item info registers
 show the cpu registers
@@ -80,8 +82,12 @@ show virtual to physical memory mappings (i386 only)
 show the active virtual memory mappings (i386 only)
 @item info hpet
 show state of HPET (i386 only)
+@item info jit
+show dynamic compiler info
 @item info kvm
 show KVM information
+@item info numa
+show NUMA information
 @item info usb
 show USB devices plugged on the virtual USB hub
 @item info usbhost
@@ -114,6 +120,10 @@ show migration status
 show balloon information
 @item info qtree
 show device tree
+@item info qdm
+show qdev device model list
+@item info roms
+show roms
 @end table
 ETEXI
 
-- 
1.7.0.3

